ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -37,10 +38,22 @@ export class ExportCommand extends Command<ExportCommandReturn> {
     });
   }
 
+  private filterPipe(pipe: EvalCommandReturn) {
+    if (!this.parameters.filter) {
+      return pipe;
+    }
+    return _.pickBy(pipe, this.parameters.filter);
+  }
+
+  private mapPipe(pipe: EvalCommandReturn) {
+    return _.mapValues(pipe, (current) => current.result.value);
+  }
+
   async run() {
     const { pipe } = this.parameters;
-    const configDict = _.mapValues(pipe, (current) => current.result.value);
-    const keyMutatedConfigDict = this.mutateKeys(configDict);
-    return keyMutatedConfigDict;
+    const filteredPipe = this.filterPipe(pipe);
+    const mappedPipe = this.mapPipe(filteredPipe);
+    const keyMutatedMappedPipe = this.mutateKeys(mappedPipe);
+    return keyMutatedMappedPipe;
```

Comment:
I don't see the value of the `mapPipe` method, unless there's a reason that I'm missing I think we should keep the code as is:
```suggestion
    const filteredPipe = this.filterPipe(pipe);
    const configDict = _.mapValues(filteredPipe, (current) => current.result.value);
    const keyMutatedConfigDict = this.mutateKeys(configDict);
    return keyMutatedConfigDict;
```